xblock/runtime.py

```diff
@@ -1211,10 +1212,22 @@ class Mixologist(object):
     """
     def __init__(self, mixins):
         """
-        :param mixins: Classes to mixin
+        :param mixins: Classes to mixin or names of classes to mixin.
         :type mixins: `iterable` of `class`
         """
-        self._mixins = tuple(mixins)
+
+        mixin_classes = []
+        for mixin in mixins:
+            if isinstance(mixin, six.text_type):
```

Comment:
I think a lot of folks will get bit by putting `['my.class.name']` in Python 2.7 without `unicode_literals` and then blowing this up because it's a byte string and not unicode.

Reply:
This is only used in one place in edx-platform for now.  But you're suggesting I use [six.string_types](https://six.readthedocs.io/#six.string_types) instead?

Reply:
Yes, but point taken that the failure scenario isn't as likely as I originally thought.